### ISSUE TITLE
In the controllers handle the semantic version validation exception as 4xx instead of 500 - internal error

### ISSUE THREAD

**Issue by tzolov** [spring-cloud/spring-cloud-skipper#484]

**Comment by markpollack** [comment]
Throw `SkipperException` and add the version that failed validation to the exception message.

**Comment by markpollack** [comment]
tested in shell.  Fixed in ece9a9cca6630b3a5539351f015e7318f278696e

```
skipper:>package upload --path /home/mpollack/skipper/helloworld-abc.zip
UploadRequest doesn't have a valid semantic version.  Version = abc
```
